lib/exporters/jhipster_application_exporter.js

```diff
@@ -79,6 +80,7 @@ function setUpApplicationStructure(application) {
 }
 
 function setUpArrayOptions(application) {
+  application[GENERATOR_NAME].blueprints = Array.from(application[GENERATOR_NAME].blueprints);
```

Comment:
I was testing this out and could not get it to work. The problem is that the `blueprints` field should be an array of objects instead of Strings. So I used the following code to fix:
```
  application[GENERATOR_NAME].blueprints = Array.from(application[GENERATOR_NAME].blueprints).map(elem => {
    return { name: elem, version: 'latest' };
  });
```

Even with this change though, I failed to use `import-jdl` command in order to init an application from scratch using multiple blueprints. Running `import-jdl` a second time uses the blueprints properly. Could not find the reason for this but I suspect it is related to how `import-jdl works`

Reply:
Haven't had the chance to properly test it out yet, but nice catch.

This behavior is weird... There must be something wrong with how blueprints are read.

Reply:
It is a bit weird. I tried to replicate the previous behavior of having `blueprint` and `bleuprintVersion` fields for a single blueprint to an array hence the resulting construct. Another way could be joining them in a single string, e.g `blueprint@version`(although we should take care for scoped blueprints as well. Any other ideas?

Reply:
I'm going to release a new version of JCore and merge JCore with the generator. If in the meantime there's a solution to this problem, I'll make a new release before the merge.

Reply:
> 
> Another way could be joining them in a single string, e.g `blueprint@version`(although we should take care for scoped blueprints as well. Any other ideas?

This is something I already thought about, but:
1. It means we should also support it from CLI : `--blueprints kotlin@1.0.0,vuejs@1.1.0`
2. It implies that generator-jhipster should install beforehand the corresponding npm packages, which is something we have never done. During the initial generator, the generator is just saving versions used for the generator.

I think it could be an interesting feature to have, but that implies a few changes in the current behavior of the generator.